addons/pronelauncher/CfgMovesMaleSdr.hpp

```diff
@@ -0,0 +1,239 @@
+            duty = 0.6;
+            disableWeapons = true;
+            actions = "ACE_LauncherProneActions";
+            interpolateTo[] = {
+                "AmovPpneMstpSrasWrflDnon", 0.02,
+                "amovppnemstpsnonwnondnon", 0.02,
+                "AmovPpneMstpSrasWpstDnon", 0.02
+            };
+        };
+        class ProneRifle_To_ProneLauncher: ProneLauncher_To_ProneRifle {
+            speed = 0.76;
+            duty = 0.6;
+            interpolateFrom[] = {
+                "AmovPpneMstpSrasWrflDnon", 0.02
+            };
+            interpolateTo[] = {
+                "AmovPpneMrunSrasWlnrDf", 0.015,
+                "AmovPpneMrunSrasWlnrDr", 0.015,
+                "AmovPpneMrunSrasWlnrDl", 0.015,
+                "ACE_LauncherProne", 0.015
+            };
+        };
+        class PronePistol_To_ProneLauncher: ProneLauncher_To_ProneRifle {
+            speed = 0.76;
+            duty = 0.6;
+            interpolateFrom[] = {
+                "AmovPpneMstpSrasWpstDnon", 0.015
+            };
+            interpolateTo[] = {
+                "AmovPpneMrunSrasWlnrDf", 0.015,
+                "AmovPpneMrunSrasWlnrDr", 0.015,
+                "AmovPpneMrunSrasWlnrDl", 0.015,
+                "ACE_LauncherProne", 0.015
+            };
+        };
+        class ProneCivil_To_ProneLauncher: ProneLauncher_To_ProneRifle {
+            speed = 0.76;
+            duty = 0.6;
+            interpolateFrom[] = {
+                "AmovPpneMstpSnonWnonDnon", 0.015
+            };
+            interpolateTo[] = {
+                "AmovPpneMrunSrasWlnrDf", 0.015,
+                "AmovPpneMrunSrasWlnrDr", 0.015,
+                "AmovPpneMrunSrasWlnrDl", 0.015,
+                "ACE_LauncherProne", 0.015
+            };
+        };
+        class AmovPercMstpSrasWlnrDnon_AmovPpneMstpSrasWlnrDnon: TransAnimBase {
+            mask = "weaponSwitching";
+        };
+        class AmovPpneMstpSrasWlnrDnon_AmovPknlMstpSrasWlnrDnon: TransAnimBase {
+            blockMobileSwitching = 0;
+            ConnectTo[] = {};
+            InterpolateTo[] += {"AmovPknlMstpSrasWlnrDnon", 0.02};
+        };
+        class LauncherProne_Reload_Start: AmovPpneMrunSrasWlnrDl {
+            actions = "ACE_LauncherProneActions";
+            speed = 0.7375;
+            duty = 0.6;
+            disableWeapons = true;
+            disableWeaponsLong = true;
+            enableMissile = false;
+            canPullTrigger = false;
+            interpolateFrom[] = {
+                "ACE_LauncherProne", 0.02
+            };
+            interpolateTo[] = {
+                "LauncherProne_Reload_Mid", 0.005
+            };
+        };
+        class LauncherProne_Reload_Mid: AmovPpneMrunSrasWlnrDr {
+            actions = "ACE_LauncherProneActions";
+            speed = 0.7375;
+            duty = 0.6;
+            disableWeapons = true;
+            disableWeaponsLong = true;
+            enableMissile = false;
+            canPullTrigger = false;
```

Comment:
There are no booleans in config. I know that Mikero auto converts them to 0 and 1 (thus breaking parsing sqf in some cases), but that seems unacceptable and I don't think it is handled by standard tools.

Reply:
>and I don't think it is handled by standard tools.
They probably turn it into string, and the number config reader will compile and execute the string as code, which will call true which will return 1. Useless performance waste here.

Reply:
For number type tokes, both `"false" and "true" are interpreted as 0.
Similar to:
```sqf
parseNumber "true" //0
```

Reply:
>parseNumber "true" //0

wiki says parseNumber true returns one.

Reply:
`getNumber`  definitely also reports `0` for `"true"` entries.

Reply:
Configs have been changed to ``0`` and ``1`` as appropriate it seems.